llvm/include/llvm/MC/MCDwarf.h

```diff
@@ -190,14 +191,19 @@ class MCDwarfLineEntry : public MCDwarfLoc {
 
 public:
   // Constructor to create an MCDwarfLineEntry given a symbol and the dwarf loc.
-  MCDwarfLineEntry(MCSymbol *label, const MCDwarfLoc loc)
-      : MCDwarfLoc(loc), Label(label) {}
+  MCDwarfLineEntry(MCSymbol *label, const MCDwarfLoc loc,
+                   bool _IsEndOfFunction = false)
+      : MCDwarfLoc(loc), Label(label), IsEndOfFunction(_IsEndOfFunction) {}
 
   MCSymbol *getLabel() const { return Label; }
 
   // This indicates the line entry is synthesized for an end entry.
   bool IsEndEntry = false;
 
+  // This indicates that the line entry belongs to a new function. To only be
```

Comment:
This is confusing. Is it the last entry of the old function or the first of the new? Should it be called `IsFunctionStart` then?

Reply:
I had originally emitted an end_sequence at the beginning of the new function, but it was easier to do it at the end of the previous function. This way there will not be an end_sequence at the start of the line table. I forgot to update the comment, I apologize for the confusion